mapNumRange.md

```diff
@@ -0,0 +1,15 @@
+### mapNumRange
+
+Maps a number from one range to another range.
+
+Returns the mapped num from the in range to the out range. 
```

Comment:
Please upgrade the snippet description, so that this line is a short explanation of the snippet's source code.

Reply:
Finding slightly difficult to explain the code. I don't what to just explain basic Math.